billing/lib/util.js

```diff
@@ -0,0 +1,15 @@
+export const startOfMonth = () => {
+  const d = new Date()
```

Comment:
I think it would be a good idea to pass seed here

```suggestion
export const startOfMonth = (now) => {
  const d = new Date(now)
```